python/xir/program.py

```diff
@@ -315,7 +324,9 @@ def options(self) -> Dict[str, Any]:
             Dict: declared scipt-level options
         """
         if self.use_floats:
-            return get_floats(self._options)
+            options_with_floats = get_floats(self._options)
+            if isinstance(options_with_floats, Dict):
+                return options_with_floats
```

Comment:
What situation are you trying to address with this `if` statement?  Isn't the result of `get_floats` always a dictionary here?

Reply:
Oh, this was just an attempt to please the static type checker. Since `options` should always return a dict, but `get_floats` returns a `Params`, which is either a dict or a list. Any recommendations for what to do here otherwise?

Reply:
Oh, I see!  Does changing the line above the `if` statement to this work?
```python
options_with_floats: Dict = get_floats(self._options)
```

Reply:
Unfortuantely, no. It complains on that new line instead (since `get_floats` doesn't always return a dict). 😞